scripts/src/models/users.ts

```diff
@@ -23,8 +30,44 @@ export class User extends CreationDateModel {
 	@OneToMany(type => OrderContext, context => context.user)
 	public contexts!: OrderContext[];
 
-	@Column({ name: "wallet_count", default: 1 })
+	@Column({ name: "wallet_count", default: User.defaultWalletCount })
```

Comment:
you can add an initializer instead (for the class):
`@Initializer("walletCount", () => User.defaultWalletCount))`
this makes more sense if you take into account my comment about the `insert.values` later on